.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: java
+jdk: oraclejdk9
+install: ./gradlew --version
+script: ./gradlew --continue --init-script gradle/init-scripts/build-scan.init.gradle.kts --scan --stacktrace sanityCheck
```

Comment:
I think we should the scans to the public instance, not to our internal instance, right?

Reply:
They should definitely go to the public instance.

Reply:
Ok, I'll change that.